deployment/src/main/java/com/datastax/oss/quarkus/deployment/internal/CassandraClientProcessor.java

```diff
@@ -64,73 +64,90 @@ FeatureBuildItem feature() {
 
   @BuildStep
   List<ReflectiveClassBuildItem> registerGraphForReflection() {
+    ReflectiveClassBuildItem.Builder builder =
+        ReflectiveClassBuildItem.builder(
+                "org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversal")
+            .constructors(true)
+            .methods(true)
+            .fields(true);
     return Arrays.asList(
         // Required for the driver DependencyCheck mechanism
-        new ReflectiveClassBuildItem(
-            true, true, "org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversal"),
+        builder.build(),
         // Should be initialized at build time:
-        new ReflectiveClassBuildItem(
-            true, true, "org.apache.tinkerpop.gremlin.tinkergraph.structure.TinkerIoRegistryV3d0"),
-        new ReflectiveClassBuildItem(
-            true, true, "org.apache.tinkerpop.shaded.jackson.databind.deser.std.StdDeserializer"),
+        builder
```

Comment:
Aren't we reusing the builder instance after a call to `build()`? This looks suspicious to me (not exactly the typical builder pattern). At least we'd need to be sure that the builder isn't accumulating data from before the last `build()` invocation.